Docs: Add usage insights export feature (#30376)
grafana/grafana: docs/sources/enterprise/usage-insights/presence-indicator.md

+++

> title = "Presence indicator"

Reviewer: How about "See who is present" instead of "Presence indicator", which sounds like a robot. We are having to work around this oddity, and let’s see what we can do.

Reviewer 2: I think we should use consistent language ("presence indicator") until we make a product-wide change.

> description = "Know who is looking at the same dashboard as you"

Reviewer: ```suggestion
description = "Know who is looking at the same dashboard as you are"
```

keywords = ["grafana", "presence-indicator", "enterprise"]
aliases = ["/docs/grafana/latest/enterprise/usage-insights/presence-indicator.md"]
weight = 300
+++

> # Presence indicator

Reviewer: ```suggestion
# See who is present
```

Reviewer 2: I think we should not change this until we change it everywhere.

> **Note:** Available in Grafana Enterprise v7.0+.

> The presence indicator is visible to all signed-in users on all dashboards. It shows the avatars of users who interacted with the dashboard recently (last 10 minutes by default). You can see the user's name by hovering your cursor over the user's avatar. The avatars come from [Gravatar](https://gravatar.com) based on the user's email.

Reviewer: ```suggestion
When you are signed in and looking at any given dashboard, you can see who is present via a presence indicator, which displays avatars of users who have interacted with the dashboard recently. The default time frame is within the past 10 minutes. To see the user's name, hover over the user's avatar. The avatars come from [Gravatar](https://gravatar.com) based on the user's email.
```

> When you are signed in and looking at any given dashboard, you can see who is present via a presence indicator, which displays avatars of users who have interacted with the dashboard recently. The default time frame is within the past 10 minutes. To see the user's name, hover over the user's avatar. The avatars come from [Gravatar](https://gravatar.com) based on the user's email.

Reviewer: ```suggestion
When you are signed in and looking at any given dashboard, you can know who is looking at the same dashboard as you are via a presence indicator, which displays avatars of users who have interacted with the dashboard recently. The default time frame is within the past 10 minutes. To see the user's name, hover over the user's avatar. The avatars come from [Gravatar](https://gravatar.com) based on the user's email.
```

> When more users are active on a dashboard than can fit in the presence indicator section, click on the `+X` icon that opens [dashboard insights]({{< relref "#dashboard-insights" >}}) to see more details about recent user activity.

Reviewer: ```suggestion
When there are more active users on a dashboard than can fit within the presence indicator, click  the **+X** icon. Doing so opens [dashboard insights]({{< relref "#dashboard-insights" >}}), which contains more details about recent user activity.
```

Reviewer: More details or more users’ details, or both?

Author: It's both. Users can see all the active users in the dashboard insights view, plus recent users and other information.

{{< docs-imagebox img="/img/docs/enterprise/presence_indicators.png" max-width="400px" class="docs-image--no-shadow" >}}

> You can choose your own definition of "recent" by setting it in the [configuration]({{< relref "../../administration/configuration.md">}}) file.

Reviewer: ```suggestion
To change _recent_ to something other than the past 10 minutes, edit the [configuration]({{< relref "../../administration/configuration.md">}}) file:
```
